feast_trino/trino.py

```diff
@@ -232,7 +245,10 @@ def _get_table_reference_for_new_entity(catalog: str, dataset_name: str) -> str:
 
 
 def _upload_entity_df_and_get_entity_schema(
-    client: Trino, table_name: str, entity_df: Union[pd.DataFrame, str],
+    client: Trino,
+    table_name: str,
+    entity_df: Union[pd.DataFrame, str],
+    connector: Dict[str, Any],
```

Comment:
If you managed to switch to a sub-model for connector, you might be able to use that as the type here, which will be more readable as well.